pkg/limayaml/limayaml.go

```diff
@@ -64,6 +64,7 @@ type Image struct {
 
 type Mount struct {
 	Location string `yaml:"location" json:"location"` // REQUIRED
+	Guest    string `yaml:"guest,omitempty" json:"guest,omitempty"`
```

Comment:
I think this should be called GuestLocation or something else.

Reply:
> I think this should be called GuestLocation or something else.

I agree. I would suggest `mountPoint`, as that is what it is called by `mount`...

Reply:
Yeah I just took the example @jandubois wrote in a related GH issue and just used that and figured we would decide on a better name in the PR. Sounds good. I'll use `mountPoint`.

Reply:
Updated